### DISCUSSION TITLE
Experiment tracking with modular, namespaced pipelines

### DISCUSSION THREAD

**Discussion by alexfurnica** [kedro-org/kedro#1359] [Idea]
First of all, thank you guys for building such an amazing open-source tool. I was just going through the Spaceflights tutorial to learn the ropes and got a little hung up on the experiment tracking section and wanted to ask for some advice.

Mainly, the issue is that the experiment tracking page of Kedro Viz has the titles of the tracked metrics, but doesn't show any actual metrics:

![image](https://user-images.githubusercontent.com/10713242/159514658-cc5ba088-297c-4424-b021-18038033b4fa.png)

This is particularly strange, since the metrics do show up just fine in the pipeline view. For example, here is the `data_science.active_modelling_pipeline.metrics` block:

![image](https://user-images.githubusercontent.com/10713242/159515145-a5d8f1bb-58e8-4f03-a873-cfd3a7de5cc5.png)

Any advice as to what I may be doing wrong?

P.S. Thought I would add this here since it is somewhat relevant. The "Tutorial" section of the documentation is a little confusing once you get to the "experiment tracking" part. Every previous section built up on the section that came before it, but the experiment tracking does not build up on the namespacing section that comes before it. If you also agree that it would be better if it did, I would be happy to contribute to the documentation.

**Comment by merelcht** [comment]
Hi @alexfurnica, thanks so much for the feedback! We have recently updated the tutorial, but since experiment tracking is a very new part we haven't fully finished making it consistent yet. I agree that we need to tidy this up.

In order to investigate the issue you're seeing with data not showing could you please confirm:
- What python version are you using?
- What version of Kedro are you using?
- What version of Kedro Viz are you using?
- What OS are you using? 

Thanks! 🙂

**Reply by alexfurnica** [reply]
Hi @MerelTheisenQB, thanks for the quick reply :) Here's the requested info:

- Python version: 3.9.7
- Kedro version: 0.17.7
- Kedro Viz version: 4.3.1
- OS: Windows 10

**Reply by merelcht** [reply]
Thanks @alexfurnica, unfortunately there's a bug in experiment tracking on Windows that's causing this. We're are working on it: https://github.com/kedro-org/kedro-viz/issues/763, so hopefully it will be fixed soon.

**Reply by alexfurnica** [reply]
Ah, thanks a lot for letting me know. Good to know you guys are already on it!

**Reply by rashidakanchwala** [reply]
this has been fixed.
